File: seed/php-model/trace/src/Submission/BuildingExecutorResponse.php
<?php

namespace Seed\Submission;

use Seed\Core\SerializableType;
use Seed\Core\JsonProperty;

class BuildingExecutorResponse extends SerializableType
{
    /**
     * @var string $submissionId
     */
    #[JsonProperty('submissionId')]
    public string $submissionId;

    /**
     * @var ExecutionSessionStatus $status
     */
    #[JsonProperty('status')]
    public ExecutionSessionStatus $status;

    /**
     * @param array{
     *   submissionId: string,
     *   status: ExecutionSessionStatus,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->submissionId = $values['submissionId'];
        $this->status = $values['status'];
    }
}

File: seed/pydantic-v2/trace/resources/submission/types/building_executor_response.py
from pydantic import BaseModel
from uuid import UUID
from resources.submission.types.execution_session_status import ExecutionSessionStatus
from dt import datetime
from core.datetime_utils import serialize_datetime
class BuildingExecutorResponse(BaseModel):
    submission_id: UUID = 
    status: ExecutionSessionStatus
    class Config:
        frozen = True
        smart_union = True
        json_encoders = {datetime: serialize_datetime}

